src/navigation/Navigation.js

```diff
@@ -13,15 +13,15 @@ const PAGES = {
   },
   '/landscapes': {
     label: 'navigation.landscapes',
-    match: path => path.startsWith('/landscapes'),
+    match: path => path === '/landscapes' || path.startsWith('/landscapes/'),
   },
   '/groups': {
     label: 'navigation.groups',
-    match: path => path.startsWith('/groups'),
+    match: path => path === '/groups' || path.startsWith('/groups/'),
   },
   '/tools': {
     label: 'navigation.tools',
-    match: path => path.startsWith('/tools'),
+    match: path => path === '/tools' || path.startsWith('/tools/'),
```

Comment:
question: why doesn't `startsWith('/tools'`)` do what we want?

Reply:
@paulschreiber `startsWith('/tools')` picks anything that starts with `/tools` but that is not exaclty what we need. The user is under or in `tools` if the route is exactly `/tools` or under `/tools/` but for example a different thing that start with `tools` like `tools123` should not be identified as `tools`, does that make sense?

Reply:
What about something like`startsWith('/tools(/.+)?')`?

Reply:
sure, a regular expression might be able to handle this too, I will test that